src/main/java/duke/duke/Deadline.java

```diff
@@ -0,0 +1,40 @@
+package duke.duke;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.time.format.TextStyle;
+import java.util.Locale;
+
+public class Deadline extends Task {
+
+    protected String dueDate;
+
+    public Deadline(String name, String dueDate) {
+        super(name);
+        this.dueDate = dueDate;
+    }
+
+    /**
```

Comment:
Not sure if there should be a space between the function and the javadoc comment of the function? based on style guide [here](https://se-education.org/guides/conventions/java/intermediate.html#comments) - think shouldn't have a space between